spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/TelemetryUtils.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.cloud.autoconfigure.telemetry;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+
+import static com.microsoft.applicationinsights.core.dependencies.apachecommons.codec.digest.DigestUtils.sha256Hex;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class TelemetryUtils {
+
+    private static final String UNKNOWN_MAC = "unknown-Mac-Address";
+
+    private static String getMacAddress() {
+        final InetAddress ip;
+        final NetworkInterface network;
+        final byte[] macBytes;
+
+        try {
```

Comment:
Could you merge declaration inside try clause, and just return in normal case?

Reply:
Each declaration and initialization could be one statement inside try clause. You can check latest change which fixed this.